apps/site/assets/ts/models/alert.ts

```diff
@@ -35,6 +35,29 @@ export const isAmenityAlert = ({ effect }: Alert): boolean =>
     "bike_issue"
   ].includes(effect);
 
+export const alertsForStopAndRoute = (
```

Comment:
Feels like there's an opportunity to combine the logic from alertsForRoute and alertsForStop here!